File: workshop1-kickoff_first_app/HelloWorldJava/app/src/main/java/com/example/helloworld_java/MainActivity.java
package com.example.helloworld_java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private EditText nameInput;
    private TextView text;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        this.nameInput = findViewById(R.id.name_input);
        this.text = findViewById(R.id.main_text);
    }

    public void updateName(View view) {
        text.setText("Hello " + nameInput.getText().toString() + " ");
    }
}

File: workshop1-kickoff_first_app/HelloWorldKotlin/app/src/main/java/com/example/helloworld/MainActivity.kt
package com.example.helloworld

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

import com.example.helloworld.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }

    fun updateName(view : View) {
        this.binding.mainText.text = "Hello ${this.binding.nameInput.text}!"
    }
}